### ISSUE TITLE
Game is running too fast, checking the box is not changing anything

### ISSUE THREAD

**Issue by Qu3nt174** [thomaswp/JunimoKartWeb#3]
Seems like checking the box doesn't change something, also submitting too.

Using on chrome ver : 126.0.6478.114

**Comment by thomaswp** [comment]
When you checked the box, did you then hit "submit"?

**Comment by Qu3nt174** [comment]
Yep, it refreshed the page and still running at 144fps (which IS my monitor frequency)

Cleaning the cookies, and set the refresh rate to 60Hz,  seems to do the trick but yeah, if i'm running in native 144hz, it hineritates the main monitor refresh rate

**Comment by thomaswp** [comment]
Got it. Hmm, not sure what's going on there. Unfortunately, that's likely an issue with the underlying rendering library, so I'm not sure I'll be able to fix it. Glad you're finding value in it, though.

**Comment by swordjjjkkk** [comment]
I submitted a fix for this issue in #7
